inc/compatibility/easy_digital_downloads.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Author:          Uriahs Victor
+ * Created on:      14/07/2021 (d/m/y)
+ *
+ * @package Neve
+ */
+namespace Neve\Compatibility;
+
+/**
+ * Class Easy_Digital_Downloads
+ *
+ * @package Neve\Compatibility
+ */
+class Easy_Digital_Downloads {
+
+	/**
+	 * Function that is run after instantiation.
+	 *
+	 * @return void
+	 */
+	public function init() {
+		if ( ! class_exists( 'Easy_Digital_Downloads' ) ) {
```

Comment:
We should add the second parameter `false` for `class_exists` so we don't autoload it. 🔄